functional-tests/steps/then.ts

```diff
@@ -0,0 +1,75 @@
+import { Then } from "cucumber";
+
+import { checkContainsText } from "@nice-digital/wdio-cucumber-steps/lib/support/check/checkContainsText";
+import { checkEqualsText } from "@nice-digital/wdio-cucumber-steps/lib/support/check/checkEqualsText";
+import { getSelector, SelectorName } from "../support/selectors";
+
+Then("I expect that the CKS GTM container is available", async () => {
+	const containerId = (await browser.executeAsync((function (
+		done: (containerId: string) => void
+	) {
+		((window as unknown) as { dataLayer: unknown[] }).dataLayer.push({
+			event: "integration-test",
+			eventCallback: function (containerId: string) {
+				done(containerId);
+			},
+		});
+	} as unknown) as () => void)) as string;
+
+	expect(containerId).toEqual("GTM-54QC4NL");
+});
+
+Then(
+	/^I expect to see "([^"]*)" in the autocomplete suggestions$/,
+	async (text: string) => {
+		const optionElement = await $(await getSelector("autocomplete option"));
+		await optionElement.waitForExist();
+
+		const menuSelector = await getSelector("autocomplete menu");
+		await checkContainsText("element", menuSelector, "", text);
+	}
+);
+
+Then(
+	/^I expect to see a meta tag named "([^"]*)" with the content "([^"]*)"$/,
+	async (name: string, content: string) => {
+		const element = await $(`meta[name='${name}']`);
+		expect(element).toHaveAttribute("content", content);
+	}
+);
+
+Then(
+	/^I expect to see a list of ([^0-9"]*)$/,
+	async (selectorName: SelectorName) => {
+		const selector = await getSelector(selectorName),
+			elements = await $$(selector);
+		expect(elements).toBeElementsArrayOfSize({ gte: 1 });
+	}
+);
+
+Then(
+	"I expect to see a list of {int} {}",
+	async (numberOfResults: number, selectorName: SelectorName) => {
```

Comment:
Noticed these steps are all using inline JS. Would it be more appropriate if these were moved into methods? Not one for this CR but something to bear in mind for the future. With the BNF end to end tests for search that's what we did to make the step files shorter and easier to read.

Reply:
It depends. I personally don't think it makes it hard to read, and if they're not reused (specific to tthis step) then I think there's little value in abstracting them out.

Reply:
Fair points! :-)